packages/babel-plugin-remove-graphql-queries/src/__tests__/index.js

```diff
@@ -9,6 +9,37 @@ function matchesSnapshot(query) {
   expect(code).toMatchSnapshot()
 }
 
+it(`Transforms queries in useStaticQuery`, () => {
+  matchesSnapshot(`
+  import React from 'react'
+  import { graphql, useStaticQuery } from 'gatsby'
+
+  export default () => {
+    const siteTitle = useStaticQuery(graphql\`{site { siteMetadata { title }}}\`)
+    
+    return (
+      <h1>{siteTitle.site.siteMetadata.title}</h1>
+    )
+  }
+  `)
+})
+
+it(`Transforms queries defined in own variable in useStaticQuery`, () => {
+  matchesSnapshot(`
+  import React from 'react'
+  import { graphql, useStaticQuery } from 'gatsby'
+
+  export default () => {
+    const query = graphql\`{site { siteMetadata { title }}}\`
+    const siteTitle = useStaticQuery(query)
+    
+    return (
+      <h1>{siteTitle.site.siteMetadata.title}</h1>
+    )
+  }
+  `)
+})
+
```

Comment:
Can we maybe add "kitchen sink" example where `StaticQuery`, page query and `useStaticQuery` is used in single file? It currently wouldn't work in gatsby as-is, but here we just test transformation correctness and not if this will work later on with gatsby

Reply:
Yeah! There’s value in at least asserting that the transform is good to go. I’ll add one!